odim-controller/odimra/group_vars/all

```diff
@@ -23,11 +23,13 @@ odim_third_party_helmcharts:
    redis: redis-5.0.8-0.1.0
    kafka: kafka-2.5.0-0.1.0
    consul: consul-1.6.0-0.1.0
+   etcd: etcd-1.16.0-0.1.0
 odim_third_party_ha_helmcharts:
    zookeeper: zookeeper-ha-2.5.0-0.1.0
    redis: redis-ha-5.0.8-0.1.0
    kafka: kafka-ha-2.5.0-0.1.0
    consul: consul-ha-1.6.0-0.1.0
+   etcd: etcd-1.16.0-0.1.0
```

Comment:
this should be etcd-ha-1.16.0-0.1.0 right?

Reply:
do we still need consul ? BB i think we have to stop the cluster and install. And this is one way upgrade unless part of initial install.

Reply:
@Bharath-KKB we will remove consul once all our services are moved from GoMicro to gRPC.

Reply:
@bharath-b-hpe Addressed the comment